File: NodeJS/browser/emulate_device.js
const { webkit, devices } = require('playwright'); 

(async () => {
  const iPhone = devices['iPhone 12'];
  const browser = await webkit.launch();
  const context = await browser.newContext({
    ...iPhone
  });

  const page = await context.newPage();
  await page.goto('https://httpbin.org/user-agent');
  const content = await page.content();
  console.log(content);

  await browser.close();
})();

File: Python/browser/emulate_device.py
from playwright.sync_api import sync_playwright

with sync_playwright() as p:
    device = p.devices["iPhone 12"] 
    browser = p.webkit.launch()
    context = browser.new_context(**device)
    page = context.new_page()
    page.goto("https://httpbin.org/user-agent")
    print(page.content())
    browser.close()
